ansible/setup.yaml

```diff
@@ -0,0 +1,8 @@
+- hosts: localhost
+  roles:
+    - role: tier4_hdr_camera_driver
+    - role: cuda
+    - role: autoware
+    - role: cyclonedds
+    - role: docker
+    - role: vcstool
```

Comment:
I personally recommend sorting roles alphabetically. If there are some dependencies, define them in `rolse/{role_name}/meta/main.yaml`.

Reply:
https://github.com/tier4/perception_ecu_container/pull/1/commits/b51f552dd8799cbb8ec74540a0ffd4a61095ac6c